mpa/modules/hooks/unlabeled_data_hook.py

```diff
@@ -2,15 +2,14 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+import importlib
 from mmcv.runner import get_dist_info
 from mmcv.runner import HOOKS, Hook
-
-from mmdet.datasets import build_dataset, build_dataloader
-
 from mpa.modules.datasets.composed_dataloader import ComposedDL
 from mpa.utils.logger import get_logger
 
 logger = get_logger()
+task_lib_name = dict(classification="mmcls", detection="mmdet", segmentation="mmseg")
```

Comment:
Great work! How about moving this hook into OTX?

Reply:
I think it's not good to move to OTX. Because it is called and defined in MPA still. We better move them to OTX after we all move stage.py and task folders.